### ISSUE TITLE
Need aria label options prop in select control

### ISSUE THREAD

**Issue by yuvrajsawant-github** [ant-design/ant-design#24331]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/iql7n)

### Steps to reproduce
Open the sandbox url and run the narrator to listen the select control option values

### What is expected?
Need Aria label prop for the option in the select control currently narrator is reading the value we need to set the custom aria label so we need this as additional property. Is there any other way to set the aria label  property to select options

### What is actually happening?
Need Aria label prop for the option in the select control currently narrator is reading the value

| Environment | Info |
|---|---|
| antd | 4.2.4 |
| React | 16.12.0 |
| System | 16.12.0 |
| Browser | Edge Version 81.0.416.72 (Official build) (64-bit) or Chorme Version 81.0.4044.138 (Official Build) (64-bit) |

**Comment by afc163** [comment]
```
<Option value="jack" aria-label="your-aria-label">Jack</Option>
```

https://codesandbox.io/s/select-with-search-field-ant-design-demo-t2cpz

**Comment by yuvrajsawant-github** [comment]
Thanks for the reply, Narrator should announce the aria-label text "your-aria-label" but it announce Value attribute "jack". Value attribute should be used to set the id like number or guid and Value is being used to set getFieldDecorator. If we set the value like number or guid then the narrator is announcing the guid not the option text "jack".

**Comment by afc163** [comment]
I don' realy understand. Could you tell me what should we do to improve select narrator behavoior?

**Comment by yuvrajsawant-github** [comment]
We are not able to set the value attribute of option as unique id like (1,2,3) because narrator is announcing the value 1 instead of option text "Jack"
<Option value="1" aria-label="your-aria-label">Jack</Option>

**Comment by afc163** [comment]
```
<Option value="1" aria-label="jack">Jack</Option>
```

**Comment by yuvrajsawant-github** [comment]
Please check the narrator it announce "1" instead of option text "Jack"

**Comment by afc163** [comment]
Reproduce: https://codesandbox.io/s/iql7n?file=/index.js

**Comment by zombieJ** [comment]
Is that mean when set `aria-label`. Option should use `aria-label` instead of `value`?

**Comment by afc163** [comment]
Is narrator reading this?

<img width="437" alt="image" src="https://user-images.githubusercontent.com/507615/82566013-e44b2980-9bad-11ea-8191-c768488dca62.png">

**Comment by yuvrajsawant-github** [comment]
Narrator is reading 1 it should read option text "Jack" not 1

**Comment by yuvrajsawant-github** [comment]
> Is that mean when set `aria-label`. Option should use `aria-label` instead of `value`?

yes correct
